src/World.cpp

```diff
@@ -2452,6 +2452,7 @@ void cWorld::BroadcastTeleportEntity(const cEntity & a_Entity, const cClientHand
 void cWorld::BroadcastThunderbolt(int a_BlockX, int a_BlockY, int a_BlockZ, const cClientHandle * a_Exclude)
 {
 	m_ChunkMap->BroadcastThunderbolt(a_BlockX, a_BlockY, a_BlockZ, a_Exclude);
+	BroadcastSoundEffect("ambient.weather.thunder", a_BlockX, a_BlockY, a_BlockZ, 50, 1);
```

Comment:
This should probably be in CastThunderbolt, not broadcast thunderbolt because the broadcasts are tied to the packets.